Panic when processing file with `import "C"` directive without comments

## What happened?
`go-ptimports` crashed with a panic when processing a file that contained an `import "C"` statement without a comment:

```go
package foo

import "C"
import (
	_ "strings"
)

func Print(s string) {
	_ = C
}
```

Adding this as a test case in the `TestPtImports` function in the `ptimports/imports_test.go` test resulted in the following panic:

```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x11e247b]

goroutine 7 [running]:
testing.tRunner.func1.2(0x12954a0, 0x14d49e0)
	/usr/local/go/src/testing/testing.go:1143 +0x332
testing.tRunner.func1(0xc000198600)
	/usr/local/go/src/testing/testing.go:1146 +0x4b6
panic(0x12954a0, 0x14d49e0)
	/usr/local/go/src/runtime/panic.go:965 +0x1b9
github.com/palantir/go-ptimports/v2/ptimports.groupImports.func1(0xc0001fa00c, 0xb, 0x8b, 0xc0001f0b50, 0xc, 0x10)
	/Volumes/git/go/src/github.com/palantir/go-ptimports/ptimports/imports.go:193 +0x5b
regexp.(*Regexp).ReplaceAllFunc.func1(0xc0001f0b50, 0xc, 0x10, 0xc0001f0b20, 0x2, 0x2, 0x0, 0x10, 0x0)
	/usr/local/go/src/regexp/regexp.go:699 +0x9e
regexp.(*Regexp).replaceAll(0xc0001f61e0, 0xc0001fa000, 0x52, 0x97, 0x0, 0x0, 0x2, 0xc000117cd8, 0xb, 0x0, ...)
	/usr/local/go/src/regexp/regexp.go:634 +0x303
regexp.(*Regexp).ReplaceAllFunc(0xc0001f61e0, 0xc0001fa000, 0x52, 0x97, 0xc000117dd0, 0x97, 0xc0001d9440, 0x0)
	/usr/local/go/src/regexp/regexp.go:698 +0xbd
github.com/palantir/go-ptimports/v2/ptimports.groupImports(0x12e1523, 0x7, 0xc000024120, 0x51, 0x87, 0xc00004c660, 0xc000024120, 0x51, 0x87, 0x0)
	/Volumes/git/go/src/github.com/palantir/go-ptimports/ptimports/imports.go:188 +0x329
github.com/palantir/go-ptimports/v2/ptimports.Process(0x12e1523, 0x7, 0xc000100600, 0x51, 0x60, 0xc00006a820, 0x14e0ca0, 0xc00004c718, 0x1059a65, 0x1073520, ...)
	/Volumes/git/go/src/github.com/palantir/go-ptimports/ptimports/imports.go:125 +0x1c5
github.com/palantir/go-ptimports/v2/ptimports_test.TestPtImports.func1(0xc000198600)
	/Volumes/git/go/src/github.com/palantir/go-ptimports/ptimports/imports_test.go:451 +0xae
testing.tRunner(0xc000198600, 0xc00000e318)
	/usr/local/go/src/testing/testing.go:1193 +0xef
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:1238 +0x2b3
```

## What did you want to happen?
`go-imports` should not crash and should operate normally.

--- a/changelog/@unreleased/pr-82.v2.yml
+++ b/changelog/@unreleased/pr-82.v2.yml
@@ -1,0 +1,5 @@
+type: fix
+fix:
+  description: Ensures that panic does not occur when processing files that contain an `import "C"` directive without comments.
+  links:
+  - https://github.com/palantir/go-ptimports/pull/82

--- a/ptimports/imports.go
+++ b/ptimports/imports.go
@@ -186,7 +186,7 @@ func groupImports(filename string, src []byte) ([]byte, error) {
 
 	cImportCommentIdx := 0
 	out = regexp.MustCompile(`\nimport "C"`).ReplaceAllFunc(out, func(match []byte) []byte {
-		if cImportCommentIdx >= len(cImportsDocs) {
+		if cImportCommentIdx >= len(cImportsDocs) || cImportsDocs[cImportCommentIdx] == nil {
 			return []byte(string(match))
 		}
 		var commentLines []string

--- a/ptimports/imports_test.go
+++ b/ptimports/imports_test.go
@@ -418,6 +418,34 @@ import "unsafe"
 ` + "`" + `
 `,
 		},
+		{
+			"CGo import without comments does not crash",
+			`package foo
+
+import "C"
+import (
+	_ "strings"
+)
+
+func Print(s string) {
+	_ = C
+}
+`,
+			&ptimports.Options{
+				Refactor: true,
+			},
+			`package foo
+
+import "C"
+import (
+	_ "strings"
+)
+
+func Print(s string) {
+	_ = C
+}
+`,
+		},
 	} {
 		t.Run(tc.name, func(t *testing.T) {
 			got, err := ptimports.Process("test.go", []byte(tc.in), tc.options)

